src/libraries/System.ServiceProcess.ServiceController/src/CompatibilitySuppressions.xml

```diff
@@ -1,9 +1,12 @@
-<?xml version="1.0"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <Suppression>
-    <DiagnosticId>CP0001</DiagnosticId>
-    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
-    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+    <DiagnosticId>CP1002</DiagnosticId>
+    <Target>System.ServiceModel.Internals.dll</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP1002</DiagnosticId>
+    <Target>SMDiagnostics.dll</Target>
```

Comment:
> Based on the diff and the amount of suppressions being added in comparison to what's being deleted, what do we gain by merging this in now instead of waiting until your mentioned change is merged and consumable?

By enabling this, we first get coverage of a current blind spot (testing facades) and we also get us closer to our goal of getting all APICompat warnings with a very small cost (which is just temporary) which is these CP1002 errors which may or may not be relevant. We already saw in other places (like your AccessControl PR) that PackageValidation has helped catch interesting issues on the build, so I'd rather check this in and have some progress made as opposed to waiting for the full solution and miss a) feedback on this feature since we are shipping it in 6.0, b) coverage for full facades, c) coverage of other potential APICompat issues with our build.

Reply:
That sounds like the right set of reasons to get this in 👍